src/test/sigopcount_tests.cpp

```diff
@@ -64,4 +65,16 @@ BOOST_AUTO_TEST_CASE(GetSigOpCount)
     BOOST_CHECK_EQUAL(p2sh.GetSigOpCount(scriptSig2), 3U);
 }
 
+
+BOOST_AUTO_TEST_CASE(blockSigOpAcceptLimit)
+{
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(0), 20000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(100), 20000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(50000), 20000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(1e6), 20000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(1e6 + 1), 40000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(2e6), 40000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(4e6 + 1), 100000);
```

Comment:
Gotcha, good to know. Any plan to try and upgrade the version of Boost used within the codebase?